src/utils/jellyfin-apiclient/getItemsHelper.ts

```diff
@@ -0,0 +1,63 @@
+import type {BaseItemDtoQueryResult} from '@jellyfin/sdk/lib/generated-client';
+import {ApiClient} from 'jellyfin-apiclient';
+
+const idsPerItemRequestLimit = 25;
+
+function getItemsSplit(apiClient: ApiClient, userId: string, options: any) {
+    const optionsTemplate = {...options};
+    const ids = options.Ids.split(',');
+    const results = [];
+
+    let nextI;
+    for (let i = 0; i < ids.length && i < options.Limit; i = nextI) {
+        nextI = i + idsPerItemRequestLimit;
+        if (nextI > options.Limit) {
+            nextI = options.Limit;
+        }
+        const idsSlice = ids.slice(i, nextI);
+        console.log(idsSlice);
```

Comment:
Was this log just included for debugging?

Reply:
Ooops, yes it was. I removed it now!